src/components/widgets/camera/services/WebrtcGortcCamera.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <video
+    ref="streamingElement"
+    autoplay
+    playsinline
+    controls
```

Comment:
We don't have `controls` on the other players, not sure we should have on this one either.

```suggestion
```

Reply:
controls is nice to have to allow us to pop out the player or cast it.. think its a value add in this case.

Reply:
In that case, only enable it in full screen mode.

Having it on the card can easily interfere with the controls we add.

Reply:
also allows for audio to be unmuted, nice to have a feature that you just listen for a print to start and then mute.  but I am okay as long.